code/__DEFINES/layers.dm

```diff
@@ -112,6 +116,9 @@
 ///Things that should render ignoring lighting
 #define ABOVE_LIGHTING_PLANE 120
 
+///Plane for sounds creating visual indicators
```

Comment:
Add dis stuff into it's own file in the skyrat defines

Reply:
I dont think it's wise to seperate layer and plane definitions